webservice/template/assignment/index.tmpl

```diff
@@ -69,7 +69,7 @@
         {{if .HasReview}}<span class="badge badge-info">Peer Review</span>{{end}}
         {{if .HasAutoValidation}}<span class="badge badge-warning">Auto Validated</span>{{end}}
 
-        <p class="lead"><strong>Deadline:</strong> {{PRETTYTIME .Assignment.Deadline}}</p>
+        <p class="lead"><strong>Deadline:</strong> {{PRETTYTIME .Assignment.Deadline}} <i>(Norwegian time!)</i></p> <!-- TODO time-norwegian -->
```

Comment:
Remove "(Norwegian time!)"

Reply:
Christopher asked for this specific, so nah